tests/helper/waf_test.cpp

```diff
@@ -64,4 +89,78 @@ TEST(WafTest, ValidRunMonitor) {
     }
 }
 
+TEST(WafTest, Logging) {
+    auto sink = std::make_shared<log_counter_sink_st>();
+    auto logger = std::make_shared<spdlog::logger>("ddappsec_test", sink);
+
+    spdlog::set_default_logger(logger);
```

Comment:
the original one should be restored at the end, otherwise the other tests will be writing to this sink, right?

Reply:
Good point.